site.yml

```diff
@@ -5,6 +5,11 @@
     - consul
 
 - hosts: mesos_masters
```

Comment:
yeh lets set this up now, so we can easily add more later. definitely seems like a thing that would go in the bootstrap scripts. That naming convention looks simple enough.